test/runtime/Policies.go

```diff
@@ -897,9 +899,24 @@ var _ = Describe("RuntimePolicies", func() {
     ]
   }
 ]`
-		_, err := vm.PolicyRenderAndImport(fqdnPolicy)
+		startPolicyRevision, err := vm.PolicyGetRevision()
+		Expect(err).To(BeNil(), "Unable to get start policy revision before policy import", err)
```

Comment:
What is "start policy revision"? This error message is unclear. Please rephrase.

Reply:
sure, I'll come up with a better name